addons/common/stringtable.xml

```diff
@@ -643,5 +643,10 @@
             <Korean>Zeus 카메라를 따라다니는 독수리의 생성 여부를 제어합니다</Korean>
             <Japanese>Zeus カメラ生成時に、イーグルがカメラを追随するかどうかを設定します。</Japanese>
         </Key>
+        <Key ID="STR_ZEN_Common_Scale">
+            <English>Scale</English>
+            <French>Redimensionnement</French>
+            <German>Skalierung</German>
+        </Key>
```

Comment:
There is no proper organization to the stringtables but I'd say move this entry up to the other single-word ones to maintain setting names and descriptions at the bottom.